MAINTAINERS.md

```diff
@@ -1,238 +1,82 @@
-# Maintainers' Guide
+# Maintaining
 
-## Branch organization
+## Overview
 
-The essence of this branching scheme is that you create "release branches" of
-independently releasable units of work.  These can then be released by their
-maintainers when ready.
+We use a three-stage release pipeline. Each stage of the release pipeline has
+its own dedicated branch and corresponding testing moon. Features and bug fixes
+progress through each stage--and are subject to testing along the way--until
+they're eventually released to the live network. This pipeline automates our
+release process, making it much easier to quickly and reliably ship code. It's
+also simple to reason about.
 
-### Master branch
+## Branches and Moons
 
-Master is what's released on the network.  Deployment instructions are in the
-next section, but tagged releases should always come from this branch.
+The branches and their corresponding moons that comprise the stages of the
+release pipeline are:
 
-### Feature branches
+--------------------------------------------------------------
+ Branch     |            Moon          | Target audience
+--------------------------------------------------------------
+`develop`   | `~binnec-dozzod-marzod`  | Kernel developers  
+`release`   | `~marnec-dozzod-marzod`  | Early Adopters  
+`release`   | `~doznec-dozzod-marzod`  | App Developers  
+`master`    | `~zod`                   | Everyone else
 
-Anyone can create feature branches.  For those with commit access to
-urbit/urbit, you're welcome to create them in this repo; otherwise, fork the
-repo and create them there.
+`~binnec` Every commit merged to develop is deployed to `binnec` 
+`~marnec` Every release branch commit is deployed to `marnec` 
+`~doznec` Late stage release candidates go to `~doznec` 
+`zod`    `zod` distributes the new kernel to the general public
```

Comment:
Small suggestion, we could incorporate the not-a-table of what gets deployed where into the table.

```suggestion
-----------------------------------------------------------------------------------
 Branch     |            Moon          | Target audience     | Contains
-----------------------------------------------------------------------------------
`develop`   | `~binnec-dozzod-marzod`  | Kernel developers   | Latest `develop` branch commit
`release`   | `~marnec-dozzod-marzod`  | Early Adopters      | Latest `release` branch commit
`release`   | `~doznec-dozzod-marzod`  | App Developers      | Latest release candidate
`master`    | `~zod`                   | Everyone else       | Latest release
```